root/etc/apparmor.d/docker_fuse

```diff
@@ -71,7 +71,9 @@ profile docker_fuse flags=(attach_disconnected,mediate_deleted) {
     /dev/{null,ptmx,tty,urandom,log} rwmlk,
     /dev/pts/* rwmlk,
 
-    /etc/{gai.conf,group,nsswitch.conf,passwd,shadow} rmkl,
+    /etc/{gai.conf,group,nsswitch.conf,shadow} rmkl,
+    # The ssh startup script does edit these files if they need modification for sshd to start
```

Comment:
Is there any way that we can have another profile for the init script?

Reply:
Like, if the init script is just another binary path, we should be able to do:

```
/titus/sshd/usr/sbin/init-titus-sshd {
... Stuff it does ...
}
```

Right?

Reply:
That is a great idea.
I can experiment with this, but is it just based on the path of the binary? does `exec` get a new profile based on the new path, or does it keep the old profile it used to have?

But yea, I think ideally I can have a second profile just for the init script, and let the next profile when `exec sshd` happens and let that take over.

Reply:
It should automatically switch profiles on exec based on path unless you're doing stuff with custom hats.